var/spack/repos/builtin/packages/source-highlight/package.py

```diff
@@ -14,12 +14,21 @@ class SourceHighlight(AutotoolsPackage, GNUMirrorPackage):
 
     homepage = "https://www.gnu.org/software/src-highlite/"
     gnu_mirror_path = "src-highlite/source-highlight-3.1.8.tar.gz"
+    git = "https://git.savannah.gnu.org/git/src-highlite.git"
```

Comment:
It's a _little_ faster and easier to (manually) confirm the following URL:

```suggestion
    git = "https://git.savannah.gnu.org/cgit/src-highlite.git"
```